### PR TITLE
Improvements for Notable changes for PS9

### PR DESCRIPTION
**Pull request by kpodemski**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 9.x
| Description?  | Improvements to a very important Notable changes page
| Fixed ticket? | n/a

Mermaid SF layout flowchart

image

### CODE REVIEW

**Comment by kpodemski** [comment]
@jolelievre could you take a look at the mermaid flowchart? :)

**Comment by jolelievre** [comment]
> @jolelievre could you take a look at the mermaid flowchart? :)

Yep seems good to me!

**Comment by kpodemski** [comment]
@jolelievre I have to switch to another task, so let's say that the first part of improvements is ready for a review

File: modules/core-updates/9.0.md [hunk 1]
```diff
@@ -323,17 +327,20 @@ if ($request) {
...
+  
+  // The session object will be automatically serialized at the end of the request,
+  // and automatically unserialized at the beginning of the next requests
   $session->setAttribute('my_custom_data', 'updated value');
 }
 ```
 
 {{% notice note %}}
-The Symfony session has other interesting features (like flash bags for unique usage data). You can read more about its usage in the <a href="https://symfony.com/doc/6.4/session.html" target="_blank">Session documentation</a>.
+he Symfony Session has other useful features, such as flash bags for storing temporary data. You can find more information about its usage in the <a href="https://symfony.com/doc/6.4/session.html" target="_blank">Session documentation</a>.
```
Comment (jolelievre) [inline modules/core-updates/9.0.md]:
```suggestion
The Symfony Session has other useful features, such as flash bags for storing temporary data. You can find more information about its usage in the <a href="https://symfony.com/doc/6.4/session.html" target="_blank">Session documentation</a>.
```

File: modules/core-updates/9.0.md [hunk 2]
```diff
@@ -388,39 +396,45 @@ We still have some work to do to replace all the usages of the legacy context wi
...
 - International > Locations > States
 - Shop parameters > Order settings > Statuses
 - Orders > Shopping Carts
-- Sell > Catalog > Attributes
-- Sell > Catalog > Feature
-- Sell > Catalog > Products (new version of the page introduce in PrestaShop V8 is now the only available one)
-  - [edition of features use a new optimized data structure in the form](https://github.com/PrestaShop/PrestaShop/pull/34763)
+- Sell > Catalog > Attributes & Features -> Attributes
+- Sell > Catalog >  Attributes & Features -> Feature
+- Sell > Catalog > Products (new version of the page introduce in PrestaShop 8.1 is now the only available one)
```
Comment (jolelievre) [inline modules/core-updates/9.0.md:413]: The carrier page should be over soon, we'll have to remember it needs to be added here

**Review by jolelievre — COMMENTED** [review]
Thanks @kpodemski 

Just one typo to fix and it can be merged

**Review by jolelievre — APPROVED** [review]
Thanks @kpodemski

**Pull request merged**